<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewWord</class>
 <widget class="QDialog" name="AddNewWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="listName">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>You cannot use the ; character. Fields with words cannot be empty, although comment field is optional. </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>You can chose background color (the default is beige).</string>
   </property>
  </widget>
  <widget class="QLabel" name="languageOneName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Language1</string>
   </property>
  </widget>
  <widget class="QLabel" name="languageTwoName">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Language2</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="languageOneEditText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="commentEditText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addWordButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>510</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Add word</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pick background color</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="beigeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Beige</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="redButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="languageTwoEditText">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QRadioButton" name="yellowButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="greenButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="grayButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Gray</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="orangeButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Orange</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="whiteButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="blueButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pinkButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>390</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Pink</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="purpleButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Purple</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
